### DISCUSSION TITLE
Ability to hide collection in Admin without having to change access

### DISCUSSION THREAD

**Discussion by denolfe** [payloadcms/payload#2357] [Feature Requests & Ideas]
There should be a way to hide collections hide a collection from admin _without having to change access control_. The only way to hide something from admin is to disable read access, but this has wider implications.

This will allow better customization of that admin experience.

**Comment by GermanJablo** [comment]
Closing as completed. Now we have `admin.hidden` ([docs](https://payloadcms.com/docs/configuration/collections#admin-options))
